workspaces/vscode-cpp/README.md

```diff
@@ -0,0 +1,3 @@
+# workspace-vscode-C/C++
+
+This image is a copy of `workspace-vscode-basic` with C/C++ installed.
```

Comment:
Can you be more specific? Which compiler (gcc/clang/etc) and so on.

```suggestion
This image is based on `prairielearn/workspace-vscode-base` with C/C++ installed.
```

Reply:
Done.